packages/opentelemetry-exporter-collector/src/platform/node/CollectorTraceExporter.ts

```diff
@@ -14,102 +14,32 @@
  * limitations under the License.
  */
 
-import { ReadableSpan } from '@opentelemetry/tracing';
-import * as grpc from 'grpc';
-import { CollectorTraceExporterBase } from '../../CollectorTraceExporterBase';
+import { ReadableSpan, SpanExporter } from '@opentelemetry/tracing';
+import { CollectorExporterNodeBase } from './CollectorExporterNodeBase';
 import * as collectorTypes from '../../types';
 
 import { CollectorProtocolNode } from '../../enums';
-import { parseHeaders } from '../../util';
-import {
-  GRPCSpanQueueItem,
-  ServiceClient,
-  CollectorExporterConfigNode,
-} from './types';
+import { CollectorExporterConfigNode, ServiceClient } from './types';
 
-import {
-  DEFAULT_COLLECTOR_URL_GRPC,
-  onInitWithGrpc,
-  sendSpansUsingGrpc,
-} from './utilWithGrpc';
-import {
-  DEFAULT_COLLECTOR_URL_JSON,
-  onInitWithJson,
-  sendSpansUsingJson,
-} from './utilWithJson';
+import { toCollectorExportTraceServiceRequest } from '../../transform';
+
+const DEFAULT_SERVICE_NAME = 'collector-exporter';
+const DEFAULT_COLLECTOR_URL_GRPC = 'localhost:55680';
```

Comment:
This is being defined at 3 places, I believe we should consolidate in one place.